### DISCUSSION TITLE
add an apline plug in before the apline load

### DISCUSSION THREAD

**Discussion by sxshateri** [laravel-frontend-presets/tall#93] [Q&A] [answered]
im trying to add the alpine-clipboard plugin to the project that I use TALL in and I have to add it before the alpine starts/loads to avoid the declaring errors in the console but since there is no alpine imports and property declare in the app.js, I'm not really sure how to do that. anyone who can help would be much appreciated.

currently by just adding the ```import Clipboard from "@ryangjchandler/alpine-clipboard"``` and ```Alpine.plugin(Clipboard)``` I get the following error in the console:
```
Uncaught (in promise) ReferenceError: $clipboard is not defined
    at eval (eval at generateFunctionFromString (module.esm.js:1)
```

**Comment by sxshateri** [comment, marked as answer]
after hours of digging found out the ```alpine js``` was imported to the ```bootstrap.js``` then it was imported to the laravel mix and i just don't get why? and most of all why there is no mention if this unorthodox way of doing things in the TALL documentation (which doesn't exist).
i literally know nobody to import ```alpine``` into ```bootstrap.js``` and then requiring it in the laravel mix because orders of requiring and importing packages matter.

**Reply by danharrin** [reply]
Feel free to PR the package and move Alpine to `app.js`.

**Reply by sxshateri** [reply]
thanks. I sure will and hope you'll accept and merge it.
